test/dynamo/test_higher_order_ops.py

```diff
@@ -1094,6 +1102,7 @@ def false_fn(x):
                     self.assertEqual(num_placeholders, 5)
 
     def test_cond_side_effect_in_one_branches(self):
+        torch._dynamo.reset()
```

Comment:
You can do this more systematically: in the setup() of the TestCase, just call torch._dynamo.reset()